app/views/notifications/_notifications_table.html.erb

```diff
@@ -0,0 +1,27 @@
+<table id="notifications" class="generic_datatable table table-striped table-bordered">
+  <% if defined? title %>
+    <caption><h3><%= title %></h3></caption>
+  <% end %>
+  <thead>
+    <tr>
+      <th>Status</th>
+      <th>Event</th>
+      <th>Start at</th>
+      <th>Channels</th>
+      <th colspan="1"></th>
+    </tr>
+  </thead>
+
+  <%# TODO: I need to allow info only Notifications %>
+  <tbody>
+    <% notifications.each do |notification| %>
+      <tr>
+        <td><%= notification.status %></td>
```

Comment:
We should add a column that is a truncation of the subject. Also, that column should be a link to the notification itself.